### DISCUSSION TITLE
How to set alignment-baseline & dominant-baseline?

### DISCUSSION THREAD

**Discussion by felicksLindgren** [vvvv/SVG#972] [Q&A]
Creating my text element here, that i want to vertically center in a rect, but can't figure out how to set alignment-baseline or dominant-baseline.
I know there's BaselinsShift implemented but that just doesn't seem right to change when i dont want to shift the baseline I want to change it.

var text = new SvgText()
            {
                Text = "help",
                TextAnchor = SvgTextAnchor.Middle,             
                Fill = new SvgColourServer(Color.Pink),
                X = new SvgUnitCollection() { new SvgUnit(50) },
                Y = new SvgUnitCollection() { new SvgUnit(50) },
                FontSize = 20,
                FontFamily = "Arial"
            };

**Comment by cantax** [comment]
There seems to be no specific property for this, but Custom Attributes might work:
`text.CustomAttributes.Add("dominant-baseline","middle")`
